code-review-chat/CodeReviewChat.ts

```diff
@@ -51,6 +52,7 @@ export class BuildChat {
 			text: message,
 			link_names: true,
 			channel: codereviewChannel.id,
+			as_user: true,
```

Comment:
This flag is often very frowned upon in slack apps and I don't think we need it. We're using a bot token and therefore all messages will be posted as the bot and will just include the PR author's name in the message. We don't actually want to post as a user themselves https://api.slack.com/methods/chat.postMessage#authorship

Reply:
As mentioned in PM, posting without `as_user` set causes the message to be attributed to "bot" with no icon, which isn't great. As we aren't actually providing a user name or icon override, this just allows the message to get the bot's name and icon